server/src/main/java/org/opensearch/extensions/ExtensionRequest.java

```diff
@@ -24,41 +25,60 @@
  */
 public class ExtensionRequest extends TransportRequest {
     private static final Logger logger = LogManager.getLogger(ExtensionRequest.class);
-    private ExtensionsManager.RequestType requestType;
+    private final ExtensionsManager.RequestType requestType;
+    private final String uniqueId;
 
     public ExtensionRequest(ExtensionsManager.RequestType requestType) {
+        this(requestType, null);
```

Comment:
Do we want to call `super` here?

Reply:
We have two constructors in this class, this way is to call another constructor in this class with null for uniqueId and it will avoid the error from having two constructors.

Reply:
@mloufra is correct here, we can't use both `super()` and `this()`.  We _could_ use `super()` in the other constructor, but the constructor for the superclass doesn't do anything so it's not needed.